catalog/templates/app-health-degraded.yaml

```diff
@@ -0,0 +1,71 @@
+message: |
+    {{if eq .serviceType "slack"}}:exclamation:{{end}} Application {{.app.metadata.name}} has degraded.
+    Application details: {{.context.argocdUrl}}/applications/{{.app.metadata.name}}.
+email:
+    subject: Application {{.app.metadata.name}} has degraded.
+slack:
+    attachments: |
+        [{
+          "title": "{{ .app.metadata.name}}",
+          "title_link": "{{.context.argocdUrl}}/applications/{{.app.metadata.name}}",
+          "color": "#f4c030",
+          "fields": [
+          {
+            "title": "Sync Status",
```

Comment:
Please copy app-health-degraded.yaml again from argocd-notifications repo to add changes of https://github.com/argoproj-labs/argocd-notifications/pull/349/files